landscape/prod/maps/redirects.map

```diff
@@ -17,3 +17,4 @@ _/kssf https://trusted.bu.edu/s/1759/2-bu/giving/interior.aspx?sid=1759&gid=2&pg
 _/sthfund https://trusted.bu.edu/s/1759/2-bu/giving/interior.aspx?sid=1759&gid=2&pgid=1333&cid=2638&dids=32&appealcode=WEBSTH ;
 _/scholarshipimpact https://www.bu.edu/campaign/ways-of-giving/annual-fund/donor-impact-page/videos/scholarship-stories/ ;
 _/celebratela http://bostonu.imodules.com/s/1759/2-bu/2col.aspx?sid=1759&gid=2&pgid=6044&content_id=6516 ;
+_/bu-wind http://www.bu.edu/sustainability/what-were-doing/bu-wind/ ;
```

Comment:
Can you add a new line at end of file to appease the hunger for new lines GitHub has?

Also, I was wondering if the target of the redirect was available over https so checked and see that the target actually responds with 404 for both.

Can you reach out to client to confirm the URL? Once confirmed and not responding with a 404 if it's available over https would set the target to https.